pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[project]
+name = "liftwing-python"
+version = "0.1.0"
+authors = [
+    {name = "Mercelis Vaughan", email = "mercelisvaughan@gmail.com"},
+]
+description = "Users will be able to retrieve JSON responses from their respective wiki APIs"
+readme = "README.md"
+requires-python = ">=3.8"
+classifiers = [
+    "Programming Language :: Python :: 3",
+]
+dependencies = [
+    "requests",
+]
+dynamic = ["version"]
```

Comment:
In order for package installation to work (e.g. `pip install -e .` this line should be removed:
We use a dynamic version when the version number isn't declared in this file but extracted from somewhere else. 
In our case we define the `version` field so if we set both we would get an error.